packages/webdoc-template-library/src/template-plugins/LinkerPlugin.js

```diff
@@ -193,7 +193,9 @@ function LinkerPluginShell() {
         return `<a href=${encodeURI(this.queryCache.get(docPath) || "")}>${linkText}</a>`;
       }
       if (isDataType(docPath)) {
-        let link = docPath.template;
+        let link = docPath.template
+          .replace(/</g, "&lt;")
+          .replace(/>/g, "&gt;");
```

Comment:
you must escape `"`, `'` and `&` then too.
ref: https://stackoverflow.com/a/40285312/8035872

Reply:
You're right. Oops.